src/functions/normTV.jl

```diff
@@ -0,0 +1,111 @@
+# Total variation norm (times a constant)
+
+export NormTV
+
+"""
+**``TV`` norm**
+
+    NormTV(λ=1)
+
+With a nonnegative scalar parameter λ, returns the function
+```math
+f(x) = λ ∑_{i=2}^{n} |x_i - x_{i-1}|.
+```
+"""
+struct NormTV{T <: Real} <: ProximableFunction
+    lambda::T
+    function NormTV{T}(lambda::T) where {T <: Real}
+        if lambda < 0
+            error("parameter λ must be nonnegative")
+        else
+            new(lambda)
+        end
+    end
+end
+
+is_separable(f::NormTV) = false
+is_convex(f::NormTV) = true
+is_positively_homogeneous(f::NormTV) = true
+
+NormTV(lambda::R=1) where {R <: Real} = NormTV{R}(lambda)
+
+function (f::NormTV)(x::AbstractArray)
+    return f.lambda * norm(x[2:end] - x[1:end-1], 1)
+end
+
+# Condat algorithm
+# https://lcondat.github.io/publis/Condat-fast_TV-SPL-2013.pdf
+function condat(y::AbstractArray, x::AbstractArray, lambda::Real)
```

Comment:
`condat` here looks like a strange name to me: maybe `tvnorm_prox_condat` is more informative?

Reply:
done